.github/workflows/swap-azure-web-app-slots.yml

```diff
@@ -145,33 +147,42 @@ jobs:
             -WebAppName ${{ inputs.app-name }} `
             -SlotName ${{ inputs.destination-slot-name }}
 
-      - name: Checkout Repository
-        if: inputs.swap-code-repo != ''
+      - name: Checkout Code Repository
+        if: inputs.app-code-repo != ''
         uses: Lombiq/GitHub-Actions/.github/actions/checkout@dev
         with:
-          repository: ${{ inputs.swap-code-repo }}
-          token: ${{ secrets.TAG_DEPLOYMENT_TOKEN }}
+          repository: ${{ inputs.app-code-repo }}
+          token: ${{ secrets.CODE_REPOSITORY_WRITE_TOKEN }}
 
       - name: Setup Git Config
-        if: inputs.swap-code-repo != ''
+        if: inputs.app-code-repo != ''
         run: |
-            git config user.email '41898282+github-actions[bot]@users.noreply.github.com'
-            git config user.name 'github-actions[bot]'
+          git config user.email '41898282+github-actions[bot]@users.noreply.github.com'
+          git config user.name 'github-actions[bot]'
+
+      - name: Remove old tags
+        uses: Lombiq/GitHub-Actions/.github/actions/remove-old-tags@issue/OSOE-861
+        with:
+          tag-prefix: ${{ inputs.tag-prefix }}
+
+      - name: Create timestamp
+        id: create-timestamp
+        uses: Lombiq/GitHub-Actions/.github/actions/create-timestamp@issue/OSOE-861
 
       - name: Fetch tags
-        if: inputs.swap-code-repo != ''
+        if: inputs.app-code-repo != ''
         run: |
             git fetch --tags
 
       - name: Determine timestamp
-        if: inputs.swap-code-repo != ''
+        if: inputs.app-code-repo != ''
         id: determine-timestamp
         run: |
             # Get the current timestamp in the format of yyyy-MM-dd-HH.mmUTC - can't use : in tags
             "timestamp=$(Get-Date -Format 'yyyy-MM-dd-HH.mmUTC')" >> $Env:GITHUB_OUTPUT
 
       - name: Delete Old prefix/latest tag
```

Comment:
Not needed anymore, right?

Reply:
Still. Since this is done in `remove-old-tags`.

Reply:
This is confusing, this step is not even in the latest version anymore.
https://github.com/Lombiq/GitHub-Actions/blob/issue/OSOE-861/.github/workflows/swap-azure-web-app-slots.yml

Reply:
It is though:

![image](https://github.com/user-attachments/assets/387c4355-52ef-4fa5-a58a-5444766f4fca)